play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -42,227 +42,226 @@ Acquired: . . . Who Needs Nano-Technologies?
+an additional +20% to unconventional maneuvers in such as well, but you and
 your vehicle take double damage for the next three turns after using these
 bonuses. (any teammates in such a vehicle do not take double damage)
 
-Kata: 
-Prerequisites: Level 10 or higher
-Whenever you land a successful melee hit against someone, you may roll for X 
-additional attacks, where X is your LVL/2. Each additional attack has 20% 
-lower chance to hit than the previous hit. (A 4th level character making their 
-3rd strike in a row would be rolling at +40% miss chance). This -20% debuff 
-applies to a monk's Shadow Strike as well, increasing the DC by 20 per strike 
-instead of increasing the strike's miss chance.
+Lucky Blind Fire: 
+Prerequisites: LCK 7
+There are always so many bullets flying around, you make a habit of not peaking your
+head out and looking at your enemy. And somehow you still hit them (some of the time).
+Roll a d10 before any Blind Firing or Hip Firing action. If the roll is from 6 to 10, 
+Gain -2 miss chance for that firing action; If the roll is from 1 to 5, nothing happens.
 
 Monk:
 Prerequisites: Level 7 or higher
 Able to use full damage for lethal strikes with unarmed.
 
 Nanite Concentration:
-Prerequisites: Level 5
-Allows the user to better concentrate on their expenditure of nanites, 
-allowing them to achieve one of the following:
-    +1 turn of cloaking
-    +20% to an combat-action at a cost of 20 nanites for 1 turn (ask GM before applying to an action. )
-    +20% to a non-combat-action at a cost of 20 nanites for 1 turn (ask GM before applying to an action.)
-    Others, run any ideas by GM.
+Prerequisites: Level 5, 20 skill in Knowledge[Nanites]
+Allows the user to better concentrate on their expenditure of nanites, allowing them
+to achieve one of the following (you must choose one option when you acquire the feat):
+    +2 turns of cloaking
+    Once per turn, +20% (or +2) to a Nanite-related-combat-action at a cost of 20 nanites (ask GM before applying to an action.)
+    Once per turn, +20% to a non-combat-action at a cost of 20 nanites (ask GM before applying to an action.)
+    Other, run any ideas by GM.
     
-Nanite Regeneration: 
-Prerequisites:
-For every crit you score, restore your nanites by the amount of crit damage 
-dealt. (10 nanites for standard crit, 20 for ex-crit)
+Nanite Absorbtion: 
+Prerequisites: 10 skill in Knowledge[Nanites]
+Once per turn, if you score a critical hit on a target that has nanites,
+the target loses up to 20 nanites and you gain the amount lost.
 
 Never Tell Me the Odds:
-Prerequisites: None
+Prerequisites: LCK 4
 Somehow you always aim at the right location, at the right time. 
-+10% Crit chance, +5% Extra-Crit chance.
++1 Critical hit range (i.e., +10% Critical Hit chance).
 
 Point-man: 
-Prerequisites:
-When clearing a room, add 10 to your reflex roll.
+Prerequisites: PER 6
+When clearing a room, add 20 to your reflex roll.

```

Comment:
Let's add some flavor text to this one. Doesn't have to be much, but let's keep it consistent.